docs/guide/object-obstacles.md

```diff
@@ -81,10 +81,10 @@ Derived from [Solid](../reference/solid.md).
 
 ```
 ThreeSteps {
-  SFVec3f    translation  0 -0.5 0
+  SFVec3f    translation  0 0 -0.5
```

Comment:
This is properly converted, but it looks strange having half of an object sank into the ground

Reply:
Yes, I think it is to allow an `increment` of 1

Reply:
Interesting :)